packages/atomic/src/components/insight/atomic-insight-tabs/atomic-insight-tabs.tsx

```diff
@@ -0,0 +1,31 @@
+import {Component, Element, h} from '@stencil/core';
+
+/**
+ *
+ * @internal
+ * @part tab - The button element for each tab.
+ */
+@Component({
+  tag: 'atomic-insight-tabs',
+  styleUrl: './atomic-insight-tabs.pcss',
+  shadow: true,
+})
+export class AtomicInsightTabs {
```

Comment:
Wonder if we could have piggybacked/re-used what Simo worked on for scrollable segmented facet. It's pretty much the same use case, and looks better than a normal scrollbar (especially on Windows)